app.js

```diff
@@ -10,17 +10,23 @@ class BookClass {
   }
 }
 
-if (gettingBooksFromLocal !== null) {
-  gettingBooksFromLocal.forEach((element) => {
-    allBooks.innerHTML += `
-    <div class="particular-book" id="${element.id}">
-      <p class="book-title-and-author">"${element.bookName}" by ${element.bookAuthor}</p>
-      <button class="remove-book" onclick="removeBook(${element.id})">Remove</button>
-    </div>
-    `;
-    const singleBook = new BookClass(element.id, element.bookName, element.bookAuthor);
-    booksObjectArray.push(singleBook);
-  });
+// if (gettingBooksFromLocal !== null) {
+gettingBooksFromLocal.forEach((element) => {
```

Comment:
- [ ] I am unable to navigate using the menu options because this `gettingBookFromLocal` is `null` for me. You can replicate this error by removing it from your localStorage completely.

  You can fix this by initializing `gettingBooksFromLocal` to an empty array if null on line 2 of this file.

  ```
  const gettingBooksFromLocal = JSON.parse(localStorage.getItem('localStorageBooks')) || [];
  ```

  ![image](https://user-images.githubusercontent.com/36542986/220817604-fff1fec1-b2d4-417a-8ac6-ea36472c7ea1.png)